src/main/webapp/WEB-INF/tags/instructor/results/participantGroupByQuestionPanel.tag

```diff
@@ -0,0 +1,55 @@
+<%@ tag description="instructorFeedbackResults - participant > question > participant" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<%@ tag import="teammates.common.util.Const" %>
+
+<%@ taglib tagdir="/WEB-INF/tags/instructor/results" prefix="results" %>
+
+<%@ attribute name="groupByQuestionPanel" type="teammates.ui.template.InstructorFeedbackResultsGroupByQuestionPanel" required="true" %>
+<%@ attribute name="shouldCollapsed" type="java.lang.Boolean" required="true" %>
+<%@ attribute name="showAll" type="java.lang.Boolean" required="true" %>
+
+
+<div class="panel ${not empty groupByQuestionPanel.questionTables ? 'panel-primary' : 'panel-default'}">
+    <div class="panel-heading">
+        ${groupByQuestionPanel.giver? 'From:' : 'To:'} 
+        <c:choose>
+            <c:when test="${groupByQuestionPanel.emailValid}">
+                <div class="middlealign profile-pic-icon-hover inline panel-heading-text" data-link="${groupByQuestionPanel.profilePictureLink}">
+                    <strong>${groupByQuestionPanel.name}</strong>
+                    <img src="" alt="No Image Given" class="hidden profile-pic-icon-hidden">
+                    <a <c:if test="${not empty groupByQuestionPanel.questionTables}">class="link-in-dark-bg"</c:if> href="mailto:${groupByQuestionPanel.participantIdentifier}">[${groupByQuestionPanel.participantIdentifier}]</a>
+                </div>
+            </c:when>
+            <c:otherwise>
+                <div class="inline panel-heading-text">
+                    <strong>${groupByQuestionPanel.name}</strong>
+                </div>
+            </c:otherwise>
+        </c:choose>
+
+        <div class="pull-right">
+            <c:if test="${groupByQuestionPanel.moderationButtonDisplayed}">
+                <results:moderationsButton moderationButton="${groupByQuestionPanel.moderationButton}" />
```

Comment:
The moderation button could be re-used in multiple places, so I think it's better to make its html classes easy to change.